pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Why did you make this change?
+(cite Issue # OR provide rationalization of changes if no issue exists)
+
+
+
+## What did you do? 
+- [add implementation details here]
```

Comment:
This might be misinterpreted and authors will describe the implementation details of their solution, which is not what we're really asking here. Maybe something like `- [describe the proposed changes]` (or work/contribution/modifications)